Source/WebKit/WebProcess/WebPage/Cocoa/UnifiedTextReplacementController.mm

```diff
@@ -288,20 +288,44 @@
     m_replacedDocumentNodes.remove(uuid);
     m_replacementLocationOffsets.remove(uuid);
 
-    removeTransparentMarkersForSession(uuid, *sessionRange);
+    removeTransparentMarkersForSession(uuid, true);
```

Comment:
This method should take in an enum parameter rather than a boolean. I can't tell what happens at the call site.